Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -156,22 +170,12 @@ define([
         optimizationPanelContents.appendChild(makeCheckbox('immediatelyLoadDesiredLevelOfDetail', 'Load only tiles that meet the max. SSE.'));
         optimizationPanelContents.appendChild(makeCheckbox('loadSiblings', 'Load siblings of visible tiles.'));
 
-        var tilesetPanel = makeSection('Tileset', 'tilesetVisible', 'toggleTileset', tilesetPanelContents);
-        var displayPanel = makeSection('Display', 'displayVisible', 'toggleDisplay', displayPanelContents);
-        var updatePanel = makeSection('Update', 'updateVisible', 'toggleUpdate', updatePanelContents);
-        var loggingPanel = makeSection('Logging', 'loggingVisible', 'toggleLogging', loggingPanelContents);
-        var tileDebugLabelsPanel = makeSection('Tile Debug Labels', 'tileDebugLabelsVisible', 'toggleTileDebugLabels', tileDebugLabelsPanelContents);
-        var stylePanel = makeSection('Style', 'styleVisible', 'toggleStyle', stylePanelContents);
-        var optimizationPanel = makeSection('Optimization', 'optimizationVisible', 'toggleOptimization', optimizationPanelContents);
-
-        // first add and bind all the toggleable panels
-        element.appendChild(tilesetPanel);
-        element.appendChild(displayPanel);
-        element.appendChild(updatePanel);
-        element.appendChild(loggingPanel);
-        element.appendChild(tileDebugLabelsPanel);
-        element.appendChild(stylePanel);
-        element.appendChild(optimizationPanel);
+        makeSection(panel, 'Tileset', 'tilesetVisible', 'toggleTileset', tilesetPanelContents);
+        makeSection(panel, 'Display', 'displayVisible', 'toggleDisplay', displayPanelContents);
```

Comment:
The `Point Cloud Shading` options under `display` used to be hidden when the box was unchecked, is that still doable? If you need it, here's some Sandcastle code for making sure everything is still linking up correctly, but all the parameters seem to work fine as of 75c731b.

```js
var viewer = new Cesium.Viewer('cesiumContainer');
viewer.extend(Cesium.viewerCesium3DTilesInspectorMixin);
var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(5713) });
viewer.scene.primitives.add(tileset);
tileset.readyPromise.then(function(){
    viewer.zoomTo(tileset);
});
```